daemon/mgr/container_rich_mode.go

```diff
@@ -0,0 +1,53 @@
+package mgr
+
+import (
+	"fmt"
```

Comment:
Please add a blank line below this line. since we have 

>  When importing packages, to improve readabilities, we should import package by sequence: system packages, project's own packages and third-party packages. And we should keep a blank line among these three kinds of packages;

In https://github.com/alibaba/pouch/blob/master/docs/contributions/code_styles.md

Reply:
ok！